services-toolkit/how-to-guides/dynamic-provisioning-rds.hbs.md

```diff
@@ -33,23 +33,67 @@ To configure dynamic provisioning for AWS RDS service instances, you must:
+- [upbound/provider-family-aws](https://marketplace.upbound.io/providers/upbound/provider-family-aws/)
 
-VMware recommends that you install the official Upbound variant.
-To install the `Provider` and to create a corresponding `ProviderConfig`, see the
-[Upbound documentation](https://marketplace.upbound.io/providers/upbound/provider-aws/latest/docs/quickstart).
+VMware recommends that you use the [upbound/provider-family-aws](https://marketplace.upbound.io/providers/upbound/provider-family-aws/) variant. This variant is both fully supported by Upbound and also allows for much more granular control over installation of APIs. This helps to alleviate performance issues often associated with with the more monolithic [upbound/provider-aws](https://marketplace.upbound.io/providers/upbound/provider-aws/). Refer to [Solving the Crossplane Provider CRD Scaling Problem with Provider Families](https://blog.crossplane.io/crd-scaling-provider-families/) to learn more.
 
-> **Important** The official documentation for the `Provider` includes a step to "Install Universal Crossplane".
-> You can skip this step because Crossplane is already installed as part of Tanzu Application Platform.
->
-> The documentation also assumes Crossplane is installed in the `upbound-system` namespace.
-> However, when working with Crossplane on Tanzu Application Platform, it is installed to the
-> `crossplane-system` namespace by default.
-> Ensure that you use the correct namespace when you create the `Secret` and the `ProviderConfig`
-> with credentials for the `Provider`.
+You will need to install both the top-level "family" Provider as well as the provider-aws-rds Provider.
+
+1. Create a file named `provider-family-aws.yaml` and copy in the
+   following contents:
+
+    ```yaml
+    # provider-family-aws.yaml
+
+    ---
+    # The AWS "family" Provider - manages the ProviderConfig for all other Providers in the same family.
+    # Does not have to be created explicitly, if not created explicitly it will be installed by the first Provider created
+    # in the family.
+    apiVersion: pkg.crossplane.io/v1
+    kind: Provider
+    metadata:
+      name: upbound-provider-family-aws
+    spec:
+      package: xpkg.upbound.io/upbound/provider-family-aws:v0.36.0
+      controllerConfigRef:
+        name: upbound-provider-family-aws
+    ---
+    # The AWS RDS Provider - just one of the many Providers in the AWS family.
+    # You can add as few or as many additional Providers in the same family as you wish.
+    apiVersion: pkg.crossplane.io/v1
+    kind: Provider
+    metadata:
+      name: upbound-provider-aws-rds
+    spec:
+      package: xpkg.upbound.io/upbound/provider-aws-rds:v0.36.0
+      controllerConfigRef:
+        name: upbound-provider-family-aws
+    ---
+    # The ControllerConfig applies settings to a Provider Pod.
+    # With family Providers each Provider is a unique Pod running in the cluster.
+    apiVersion: pkg.crossplane.io/v1alpha1
```

Comment:
**thought**

I get 
```
Warning: This API is deprecated and is scheduled to be removed in a future release.
controllerconfig.pkg.crossplane.io/upbound-provider-family-aws created
``` 

But I recall that they actually had no plans to deprecate this as there is no alternative. But is this still their recommended model to use provider families?

Reply:
Yeah I thought that as well, but it's still what is being recommended: https://docs.upbound.io/upbound-marketplace/providers/#using-controllerconfigs